reference/dic_tags.rst

```diff
@@ -711,7 +711,7 @@ kernel.reset
 
 **Purpose**: Clean up services between requests
 
-During the ``kernel.terminate`` event, Symfony looks for any service tagged
+When it's not the first Request, Symfony looks for any service tagged
```

Comment:
Yes, this is a bit hard to understand. What about being a bit more explicit to also mention main/sub requests, etc.

Proposal:

```rst
In all main requests (not :ref:`sub-requests <http-kernel-sub-requests>`) except
the first one, Symfony looks for any service tagged with the ``kernel.reset`` tag
to reinitialize their state. This is done by calling to the method whose name is
configured in the ``method`` argument of the tag.
```

Reply:
I applied your suggestion :smiley: Thanks